### PR TITLE
zh: stop tidb gracefully

### PR DESCRIPTION
**Pull request by weekface**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/access-tidb.md [hunk 1]
```diff
@@ -65,3 +65,54 @@ kubectl -n ${namespace} get svc ${cluster_name}-tidb -ojsonpath="{.spec.ports[?(
...
 访问 [Kubernetes Service 文档](https://kubernetes.io/docs/concepts/services-networking/service/)，了解更多 Service 特性以及云平台 Load Balancer 支持。
+
+## 平滑升级 TiDB 集群
+
+滚动更新 TiDB 集群的过程中，在停止 TiDB Pod 之前，Kubernetes 会向 TiDB server 进程发送一个 [`TERM`](https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods) 信号。在收到 `TERM` 信号后，TiDB server 会尝试等待所有的连接关闭，不过 15 秒后会强制关闭所有连接并退出进程。
+
+从 v1.1.2 版本开始，TiDB Operator 已经支持平滑升级 TiDB 集群了。通过配置下面两个属性来实现平滑升级 TiDB 集群：
+
+- `spec.tidb.terminationGracePeriodSeconds` 滚动更新的时候，删除旧的 TiDB Pod 最多容忍的时间，即过了这个时间，TiDB Pod 会被强制删除；
+- `spec.tidb.lifecycle` 设置 TiDB Pod 的 `preStop` Hook，在 TiDB server 停止之前执行的操作。
```
Comment (ran-huang) [inline zh/access-tidb.md:76]:
```suggestion
- `spec.tidb.terminationGracePeriodSeconds`：滚动更新的时候，删除旧的 TiDB Pod 最多容忍的时间，即过了这个时间，TiDB Pod 会被强制删除；
- `spec.tidb.lifecycle`：设置 TiDB Pod 的 `preStop` Hook，在 TiDB server 停止之前执行的操作。
```

File: zh/access-tidb.md [hunk 2]
```diff
@@ -65,3 +65,54 @@ kubectl -n ${namespace} get svc ${cluster_name}-tidb -ojsonpath="{.spec.ports[?(
...
+    lifecycle:
+      preStop:
+        exec:
+          command:
+          - /bin/sh
+          - -c
+          - "sleep 10 && kill -QUIT 1"
+```
+
+- 设置了删除 TiDB Pod 的最多容忍时间为 60 秒，如果 60 秒之内客户端仍然没有关闭连接的话，那么这些连接将会强制关闭。这个时间可根据需要进行调整；
```
Comment (ran-huang) [inline zh/access-tidb.md:115]:
```suggestion
上述 YAML 文件中：

- 设置了删除 TiDB Pod 的最多容忍时间为 60 秒，如果 60 秒之内客户端仍然没有关闭连接的话，那么这些连接将会强制关闭。这个时间可根据需要进行调整；
```

File: zh/access-tidb.md [hunk 3]
```diff
@@ -65,3 +65,56 @@ kubectl -n ${namespace} get svc ${cluster_name}-tidb -ojsonpath="{.spec.ports[?(
 若运行在有 LoadBalancer 的环境，比如 GCP/AWS 平台，建议使用云平台的 LoadBalancer 特性。
 
 访问 [Kubernetes Service 文档](https://kubernetes.io/docs/concepts/services-networking/service/)，了解更多 Service 特性以及云平台 Load Balancer 支持。
+
+## 平滑升级 TiDB 集群
```
Comment (DanielZhangQD) [inline zh/access-tidb.md:69]: I think we should move this section to the `configure-cluster-using-tidbcluster.md` file, users should configure this when they're configuring the TidbCluster. WDYT? @ran-huang @weekface

Comment (lilin90) [reply zh/access-tidb.md:69]: @weekface PTAL~

Comment (weekface) [reply zh/access-tidb.md:69]: 👌，等这个 PR merge 了一起修改：https://github.com/pingcap/docs-tidb-operator/pull/544

**Pull request merged**